app/javascript/components/maintaining_iterations_summary_table.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
```

Comment:
I suggest namespacing this file into `components/maintaining/iterations_summary_table`

Reply:
I can do that. I didn't do that because `application.jsx` said:

> // Add all react components here.
> // Each should map 1-1 to a compoenent in app/helpers/components

Reply:
Sweet. I think it's the right thing to do, and then we need to work out how to do the actual components at the Ruby side. I think I want to make those as actually classes somehow, not just a million arbitary methods in that helper file (I tend to hate helpers)

Reply:
I am not a fan of helpers either